### PR TITLE
refactor: use grid layout for pagination nav

### PR DESCRIPTION
**Pull request by lex111**
- Less code
- Simpler markup
- No need for adding an empty element if there is no next/prev link

### CODE REVIEW

File: packages/core/demo/index.html [hunk 1]
```diff
@@ -2163,63 +2163,62 @@ <h1>
...
           </div>
           <div class="row">
             <div class="col col--5">
               <nav class="pagination-nav">
-                <div class="pagination-nav__item">
-                  <a class="pagination-nav__link" href="#url">
-                    <div class="pagination-nav__sublabel">Previous</div>
-                    <div class="pagination-nav__label">Installation</div>
-                  </a>
-                </div>
```
Comment (Josh-Cena) [inline packages/core/demo/index.html:2209]: Is this change necessary, or is it just an optimization?

Comment (Josh-Cena) [reply packages/core/demo/index.html:2209]: Oh... I see that `.pagination-nav__item` doesn't exist anymore

Comment (lex111) [reply packages/core/demo/index.html:2209]: Just a slight improvement.

**Comment by Josh-Cena** [comment]
Noticed a little inconsistency we had previously:

Screen Shot 2022-04-25 at 3 44 10 PM

This change looks like an improvement 👍
